<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>861</width>
      <height>60</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Dictionary</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Thesaurus</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Games</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Language to Learn</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>879</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Language"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionSpanish"/>
    <addaction name="actionCebuano"/>
    <addaction name="actionIrish"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSpanish">
   <property name="icon">
    <iconset resource="languagesIcons.qrc">
     <normaloff>:/languages/languages/Spain-Flag.png</normaloff>:/languages/languages/Spain-Flag.png</iconset>
   </property>
   <property name="text">
    <string>Spanish</string>
   </property>
  </action>
  <action name="actionCebuano">
   <property name="icon">
    <iconset resource="languagesIcons.qrc">
     <normaloff>:/languages/languages/Philippines-Flag-icon.png</normaloff>:/languages/languages/Philippines-Flag-icon.png</iconset>
   </property>
   <property name="text">
    <string>Cebuano</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionIrish">
   <property name="icon">
    <iconset resource="languagesIcons.qrc">
     <normaloff>:/languages/languages/irish_flag.jpg</normaloff>:/languages/languages/irish_flag.jpg</iconset>
   </property>
   <property name="text">
    <string>Irish</string>
   </property>
  </action>
  <action name="actionNew_Language">
   <property name="text">
    <string>New Language</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="languagesIcons.qrc"/>
 </resources>
 <connections/>
</ui>
